packages/native/test/lib/ElementAssertion.test.tsx

```diff
@@ -271,4 +271,64 @@ describe("[Unit] ElementAssertion.test.ts", () => {
       });
     });
   });
+
+  describe (".toContainElement", () => {
+    const element = render(
```

Comment:
Destructiring helps to make the queries simpler:
```suggestion
    const { getByText, ... } = render(
```